docs/cypress/data/validatedLinks.ts

```diff
@@ -7,6 +7,8 @@
  */
 
 export const VALIDATED_LINKS = [
+  'https://docs.aws.amazon.com/amplify/latest/userguide/getting-started.html',
+  'https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-attributes.html',
```

Comment:
Those might be blocked by some sort of WAF since we got 403 errors from NodeJS request